samigo/samigo-app/src/webapp/jsf/evaluation/gradeStudentResult.jsp

```diff
@@ -174,9 +174,9 @@ function toPoint(id)
   <fieldset class="short-summary-box">
     <legend class="summary-title"><h:outputText value="#{evaluationMessages.summary_title}" rendered="true" /></legend>
     <ul>
-      <li><h5 style="display: inline;"><h:outputText value="#{evaluationMessages.correct_title}" />: <h:outputText value="#{totalScores.results[studentScores.studentId][0]}" /></h5></li>
-      <li><h5 style="display: inline;"><h:outputText value="#{evaluationMessages.incorrect_title}" />: <h:outputText value="#{totalScores.results[studentScores.studentId][1]}" /></h5></li>
-      <li><h5 style="display: inline;"><h:outputText value="#{evaluationMessages.empty_title}" />: <h:outputText value="#{totalScores.results[studentScores.studentId][2]}" /></h5></li>
+      <li><h5 style="display: inline;"><h:outputText value="#{evaluationMessages.correct_title}" />: <h:outputText value="#{totalScores.results[studentScores.getLongAssessmentGradingId()][0]}" /></h5></li>
```

Comment:
Does this work ? Sounds weird to me, shall it be **studentScores.longAssessmentGradingId**

Reply:
Yes, it works, but if you want i can use a variable

Reply:
Does the variable work ? You can try without restarting the server

Reply:
No, it doesn't work, to do that it will need a new variable

Reply:
okay, so let's commit this as is ?

Reply:
For me yes

Reply:
Sorry for the mistake it can be studentScores.longAssessmentGradingId without the getter. Now it is correct